Documentation/rkt-vs-other-projects.md

```diff
@@ -51,6 +61,18 @@ As rkt does not have a centralized daemon it can also be easily integrated with
 [runc-github]: https://github.com/opencontainers/runc
 [oci-spec-github]: https://github.com/opencontainers/specs
 
+## rkt vs containerd
+
+[containerd][containerd-github] is a daemon to control runC.
+It has a command-line tool called `ctr` which is used to interact with the containerd daemon.
+This makes the containerd process model similar to that of the Docker process model, illustrated above.
+
+Unlike the Docker daemon it has a reduced feature set; not supporting image download, for example.
+
+rkt has no centralized daemon to manage containers, instead launching containers directly from client commands, making it compatible with init systems such as systemd, upstart, and others.
```

Comment:
Unrelated to this PR, but I ponder about removing "*, upstart, and others*" here and in similar sentences in other paragraphs: in practice, we have not done the work of testing & making sure rkt can run as services with upstart and "others".